tests/test_bench.py

```diff
@@ -70,15 +73,18 @@ def sample_filenames() -> list[str]:
         "ae_indep_qiskit_10.qasm",
         "ghz_nativegates_rigetti_qiskit_opt3_54.qasm",
         "ae_indep_tket_93.qasm",
+        "ae_indep_bqskit_93.qasm",
```

Comment:
Maybe a smaller circuit would be better suited for BQSKit because of their runtime (especially for optimization levels >2)